RootTools/python/DataMC/Histogram.py

```diff
@@ -49,6 +49,12 @@ def __str__(self):
                          unweighted = self.Yield(weighted=False) )
         return tmp
 
+    def uwYield(self):
```

Comment:
What's the advantage of having these two, in particular since they have no documentation lines?